versioned/persist/nontx/src/main/java/org/projectnessie/versioned/persist/nontx/NonTransactionalDatabaseAdapter.java

```diff
@@ -1022,7 +1022,11 @@ protected final RefLogParents fetchRefLogParents(
   }
 
   protected final int refLogStripeForName(String refName) {
-    return Math.abs(refName.hashCode()) % config.getRefLogStripes();
+    int h = refName.hashCode();
+    if (h == Integer.MIN_VALUE) {
+      h++;
```

Comment:
Aren't we breaking backward compatibility at storage level with this change to stripe computation?

Reply:
Nope. If the hash was ever `MIN_VALUE`, if would have errored out

Reply:
Not sure about errors... I could not find a solid error path by reading the code :thinking:  Most paths lead to constructing database keys from the integer and those look fine wrt negative numbers :shrug: Nonetheless, I guess in general the probably of hitting this hash value is low and reflog is deprecated... so it's probably of to do this change.